src/search/sr_optimizers.c

```
@@ -0,0 +1,331 @@
+  if (!raw || !out_value) {
+    return 0;
+  }
+  parsed = strtol(raw, &end, 10);
+  if (end == raw || parsed <= 0 || parsed > INT_MAX) {
+    return 0;
+  }
+  *out_value = (int)parsed;
+  return 1;
+}
+
+static int sr_optimizer_read_seed_env(const char *name, uint64_t *out_value)
```

Clang (14.0.6)

[warning] parameter 'out_value' is unused

(misc-unused-parameters)